src/VisualStudio/IntegrationTest/TestUtilities/WellKnownGlobalOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Options;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities
+{
+    /// <summary>
+    /// Options settable by integration tests.
+    /// 
+    /// TODO: Options are currently explicitly listed since <see cref="OptionKey"/> is not serializable.
+    /// https://github.com/dotnet/roslyn/issues/59267
```

Comment:
💡 I recommend resolving this as _won't fix_. New integration tests already support `OptionKey` directly, and old integration tests are being actively ported to the new library.